<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>65</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
        background-color:#fdf1d9;
        font-family:Poiret One;
        font: bold;
    }

QPushButton{
        background-color: #ffc05c;
    border-style: solid;
    border-width: 2px;
    border-radius: 17px;
    border-color: #ffc05c;
    font: bold 18px;
    padding: 8px;
	color:#2584a7;
    
    }
    
    QPushButton:hover{
        border-color: #fe6d73;
    }
    
    QPushButton:pressed,  QPushButton:checked{
        border-width: 5px;
        border-radius: 15px;
        font:  15px;
        border-color: #fe6d73;
    }

    QDateEdit {
        color: #1a5d75;
        font: bold 18px;
    }
    
    QSpinBox {
        background-color: #36a9d3;
        color: #1a5d75;
        border: 1px solid #36a9d3;
        border-radius: 5px;
    }
    QSpinBox::up-button{
        border:none;
        image: url(file:///../UI/images/icon_up_arrow.png);
    }
    QSpinBox::down-button{
        border:none;
        image: url(file:///../UI/images/icon_down_arrow.png);
    }

QLabel {
		color: #1a5d75;
}

QSlider::groove:horizontal {
		background-color: #ffe0ad;
        color: #1a5d75;
        border: 2px solid #ffe0ad;
    /*border: 1px solid #1a5d75;*/
    	height: 18px;
    /*background: #ffc05c;*/
    	margin: 0px;
    	border-radius: 12px;
		padding: 2px;
	}
	QSlider::handle:horizontal {
    	background: #1a5d75;
    	border: 0px solid #1a5d75;
    	width: 70px;
    	border-radius: 9px;
	}

	QSlider::sub-page:horizontal{
		background-color: #36a9d3;
        color: #1a5d75;
        border: 2px solid #36a9d3;
    /*border: 1px solid #1a5d75;*/
    	height: 18px;
    /*background: #ffc05c;*/
    	margin: 0px;
    	border-radius: 12px;
		padding: 2px;
	}

QCalendarWidget QToolButton {
  	height: 60px;
  	width: 150px;
  	color: #1a5d75;
  	font-size: 20px;
  	icon-size: 20px, 20px;
  	background-color: #ffc05c;
	border:none;
  }
  QCalendarWidget QMenu {
  	width: 150px;
  	left: 20px;
  	color: #36a9d3;
  	font-size: 20px;
  	background-color: #ffe0ad;
  }
  QCalendarWidget QSpinBox { 
  	width: 150px; 
  	font-size:20px; 
  	color: white; 
  	background-color: #ffc05c;
  	selection-background-color: #ffe0ad;
  	selection-color: #1a5d75;
  }
  QCalendarWidget QSpinBox::up-button { subcontrol-origin: border;  subcontrol-position: top right;  width:30px; }
  QCalendarWidget QSpinBox::down-button {subcontrol-origin: border; subcontrol-position: bottom right;  width:30px;}
  QCalendarWidget QSpinBox::up-arrow { width:30px;  height:30px; }
  QCalendarWidget QSpinBox::down-arrow { width:30px;  height:30px; }
   
  /* header row */
  QCalendarWidget QWidget { alternate-background-color: #ffc05c; }
   
  /* normal days */
  QCalendarWidget QAbstractItemView:enabled 
  {
  	font-size:20px;  
  	color: #ffe0ad;  
  	background-color: #36a9d3;  
  	selection-background-color: #ffe0ad; 
  	selection-color: #17c3b2; 
  }
   
  /* days in other months */
  /* navigation bar */
QCalendarWidget QWidget#qt_calendar_navigationbar
{ 
  background-color: #ffe0ad; 
}

QCalendarWidget QAbstractItemView:disabled 
{ 
color: #1a5d75; 
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="3" column="0" colspan="3">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="10">
       <widget class="QDateEdit" name="date_to">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="displayFormat">
         <string>dd-MM-yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>du</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="11">
       <widget class="QSlider" name="slider">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>14</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6">
       <widget class="QDateEdit" name="date_from">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="displayFormat">
         <string>dd-MM-yyyy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>au</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pB_from">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Date de début</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>images/icon_date_2.png</normaloff>images/icon_date_2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
